test/lit/passes/outlining.wast

```diff
@@ -1006,3 +1006,57 @@
     (loop (nop))
   )
 )
+
+;; Test that no attempt is made to outline overlapping repeat substrings
+(module
+  (func $a
+    (drop (i32.add
+      (i32.const 0)
+      (i32.const 1)
+    ))
```

Comment:
We could make the test more concise and easy to read by just using constants and drops, unless I'm missing some reason why this wouldn't work.
```
(drop (i32.const 0))
(drop (i32.const 1))
(drop (i32.const 2))
(drop (i32.const 3))
(drop (i32.const 0))
(drop (i32.const 1))
(drop (i32.const 2))
(drop (i32.const 3))
(drop (i32.const 1))
(drop (i32.const 2))
(drop (i32.const 1))
(drop (i32.const 2))
```

Reply:
good call, thanks